lib/components/DepositForm.jsx

```diff
@@ -47,15 +45,15 @@ export const DepositForm = (props) => {
   let overBalance = false
   try {
     depositAmountBN = ethers.utils.parseUnits(depositAmount || '0', tokenDecimals)
-    overBalance = depositAmountBN && usersTokenBalance && usersTokenBalance.lt(depositAmountBN)
+    overBalance =
+      depositAmountBN &&
+      usersTokenBalanceUnformatted &&
+      usersTokenBalanceUnformatted.lt(depositAmountBN)
   } catch (e) {
     console.warn(e)
   }
 
-  const tokenBal =
-    usersTokenBalance && tokenDecimals
-      ? ethers.utils.formatUnits(usersTokenBalance, tokenDecimals)
-      : ''
+  const tokenBal = usersTokenBalance
```

Comment:
Is this var necessary? Prob can switch `tokenBal` below with `usersTokenBalance` instead.